src/map/map.ts

```diff
@@ -346,6 +342,14 @@ export class PropertiesMap {
         this._settingsModal.remove();
     }
 
+    public get active() {
```

Comment:
```suggestion
    public get active(): string {
```

This needs documentation